.github/integration/tests/sda/20_ingest-verify_test.sh

```diff
@@ -69,4 +69,11 @@ until [ "$(curl -su guest:guest http://rabbitmq:15672/api/queues/sda/verified/ |
     sleep 2
 done
 
-echo "ingestion and verification test completed successfully"
+# check that the files have key hashes assigned
+key_hashes="$(psql -U postgres -h postgres -d sda -At -c "select distinct key_hash from sda.files" | wc -l)"
+if [ "$key_hashes" -lt 0 ]; then
```

Comment:
```suggestion
if [ "$key_hashes" -eq 0 ]; then
```

Reply:
The result of `wc -l` can never be negative.

Reply:
Very good point :see_no_evil: . Fixed in 6f8ebfa.